pandora-common/src/assets/appearanceValidation.ts

```diff
@@ -60,6 +60,17 @@ export type AppearanceValidationResult = {
 	error: AppearanceValidationError;
 };
 
+export function AppearanceItemsCalculateTotalCount(items: AppearanceItems): number {
+	let itemCount = 0;
+	for (const item of items) {
+		itemCount++;
+		for (const module of item.getModules().values()) {
+			itemCount += AppearanceItemsCalculateTotalCount(module.getContents());
+		}
+	}
+	return itemCount;
+}
```

Comment:
Nit: I feel this function makes more sense to be put in `./validation.ts` file. Also avoids cyclic import.